scss/4-elements/_forms.scss

```diff
@@ -211,8 +211,11 @@ textarea,
   }
 }
 
-.cads-form-optional {
-  .cads-form-label::after {
+.cads-form-label {
```

Comment:
This is the 🐛 fix. Makes labels block, and accounts for reduced margin when a hint is present 👇